app/views/foreman_plugin_template/layouts/react.html.erb

```diff
@@ -0,0 +1,16 @@
+ 
+<% content_for(:javascripts) do %>
+  <%= webpacked_plugins_js_for :'foreman-tasks' %>
+<% end %>
+<% content_for(:stylesheets) do %>
+  <%= webpacked_plugins_css_for :'foreman-tasks' %>
+<% end %>
+
+<% content_for(:content) do %>
+  <%= notifications %>
+  <div id="organization-id" data-id="<%= Organization.current.id if Organization.current %>" ></div>
```

Comment:
Should `ForemanContext` be used instead?

Reply:
Sounds like a nice to have :+1:  , but it needs a PR in foreman-core for adding taxonomies into `app_metadata`

Reply:
I've added taxonomies and user_id to the app_metadata here - https://github.com/theforeman/foreman/pull/7655